roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,241 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | oo_selector_to_string_list() }}"
+    description: Prometheus
+
+# secrets
+- name: Set alert and prometheus secrets
+  oc_secret:
+    state: "{{ state }}"
+    name: "{{ item }}-proxy"
+    namespace: "{{ openshift_prometheus_namespace }}"
+    contents:
+      - path: session_secret
+        data: "{{ 43 | oo_random_word }}="
+  with_items:
+    - prometheus
+    - alerts
```

Comment:
@zgalor I think this should have been:

```yaml
  with_items:
    - prometheus
    - prometheus-alerts
```
Are we still in time to fix it?

Reply:
This was intentional, to align with the template. 
Naming is not 100% unified but at least it is consistent. 
Only the configmap and service certificate secret are called `prometheus-alerts`. 
All other entities (service, route, proxy secrets) are called `alerts`.

I see we have a cert sign error from the service that doesn't appear when using the template. I am investigating this